symphony/bdk/core/config/model/bdk_rsa_key_config.py

```diff
@@ -19,3 +19,36 @@ def is_valid(self) -> bool:
         :return: true if the RSA configuration is valid.
         """
         return not (self.path is not None and self.content != "")
+
+    def set_content(self, rsa_key_content):
+        """Sets rsa content and overrides path to None to keep key's config valid
+        Args:
+            rsa_key_content: rsa private key content
+        """
+        self.content = rsa_key_content
+        self.path = None
+
+    def set_path(self, rsa_key_path):
+        """Sets rsa path and overrides content to None to keep key's config valid
+        Args:
+            rsa_key_path: rsa private key path
+        """
+        self.path = rsa_key_path
+        self.content = None
+
+    def get_private_key_from_config(self):
+        """Loads the private key content.
+        If the path is set, it loads the file content, otherwise it returns the content.
+        Returns: private key content as string.
```

Comment:
I saw you used the `:return` (reStructuredText) everywhere 👌 
can you please add a lowercase R here